### ISSUE TITLE
oauthLogin.html can't be found

### ISSUE THREAD

**Issue by shcherbachev** [pwielgolaski/teamcity-oauth#7]
Hi !

I've installed 1.0.4 version of this plugin into my my local TeamCity instance v10.0.5 (build 42677). After adding the OAuth module and configuring it the new option appeared on the login screen.
However then I press on it I see an error:

> No webpage was found for the web address: http://servername/oauthLogin.html?submitLogin=Log+in+via+oAuth
> HTTP ERROR 404

How can I fix this?

**Comment by pwielgolaski** [comment]
Do you use docker version?
I haven't seen errors like that.
What is your operating system?

**Comment by shcherbachev** [comment]
TeamCity is running on Windows Server 2012 inside of Virtual Box VM. 
We plan to migrate it to a docker although.

**Comment by pwielgolaski** [comment]
I checked latest version on Windows Azure (Windows 2012) - all seems to work fine.
If issue persist, I would need more information to help.

**Comment by miketrionfo** [comment]
Hello, @shcherbachev  I encountered the same problem today.  My problem was that I had listed multiple scopes in the OAuth scope option `email profile` and the url was not encoded on the way over to the OAuth provider resulting in the following exception (found in log/teamcity-winservice.txt) because a space is not a legal url character:

`java.lang.IllegalArgumentException: Illegal character in query at index 160`

In order to get it to work I had to manually escape my scope options to `email%20profile` and then it worked fine. 

@pwielgolaski If you decide to escape it, let me know because the update would doubly escape my scope breaking my authentication.  Thanks!
